### ISSUE TITLE
Flayed Demon Skin Will Not Drop In Desolace

### ISSUE THREAD

**Issue by scaryghost138** [elysium-project/server#2322]
I've been trying to get Flayed Demon Skin to drop from the mobs around Thunder Axe Fortress in Desolace for a couple hours and I have not seen it drop.  Websites report a 2% drop rate among all the mobs around the fortress making me believe I should statistically have seen it by now and I have asked other people and not talked to anyone that has seen it dropped.  I was wondering if this could be looked into because I'm farming Undercity rep and this begins a quest chain that could net me 750 rep.  Thank you for your time and effort towards Elysium, 
Scaryghost

**Comment by scaryghost138** [comment]
The quest chain name is "The Corruptor" if that helps in any way.

**Comment by Maduse** [comment]
I've had it dropped recently.

**Comment by Marcelo20XX** [comment]
The chance for the item is correctly at 2% on the database, so this is not a valid issue. In the future please consider doublechecking item's chance to dropdirectly from the Elysium SQL database hosted in their GitHub first before reporting something...

**Comment by thepparker** [comment]
The drop is added at a later stage of the game than the patch Elysium is currently running.

**Comment by Bloontede** [comment]
https://web.archive.org/web/20050508144052/http://wow.allakhazam.com:80/db/quest.html?wquest=1481

If you look at the comments you'll see that the quest was actually there already back in 2004, so this isn't true. The skin should drop on elysium.

I know the drop is supposed to be rare (took me 100+ kills a couple months ago). That said, I just killed ~1000 mobs with 4 other dudes for 6h+ and didn't see it. I think at this point it's pretty fair to assume something is up (the database on github is 1 week old, seems like the change was done 3-4 days ago).

Please re-open.

**Comment by brotalnia** [comment]
I believe the confusion comes from the fact that the original item was replaced with a new one to prevent alliance characters from taking the quest.

http://db.vanillagaming.org/?item=6766
http://db.vanillagaming.org/?item=20310

**Comment by TomSesselmann** [comment]
Any updates on this?

**Comment by Jakobud** [comment]
> The drop is added at a later stage of the game than the patch Elysium is currently running.

I have had it drop on characters when I was on the Zeth'kur server. Is something different about the patch version after the Elysium xfer?

**Comment by gearselysium** [comment]
@Jakobud the dropping of this item was fixed as it should not be dropping yet.

**Comment by Jakobud** [comment]
FYI on my latest character I farmed these guys for abotu 100+ kills and no drop. On other characters when I was back on Zeth'kur I got it to drop within 30 kills.

**Comment by Disdodolphin** [comment]
When I initially leveled when this server just got released, I had it drop. Now releveling it just can't drop. I've farmed for 5~ hours and asked several people in general chat and people leveling there and I haven't seen a single person stating they've gotten it to drop.

Something must've messed with this drop, be that patches between launch and now or something else is hard to determine, but unless everybody around me is incredibly unlucky, it's bugged.
